README.md

```diff
@@ -1 +1,89 @@
-# tic-tac-toe
\ No newline at end of file
+#  Tic Tac Toe OOP build with Ruby (Object Oriented Programming)
+* Original project specification at [The Odin Project] (https://www.theodinproject/lesseons/oop)
+* The purpose of the project is to master object oriented programming in ruby.
+
+# This project has four milestones
+- Project setup
+- User interface
+- Game logic
+- Instruction for players
+
+
+# Game Rules:
+Tic Tac Toe is Cool Math Game:
+Each player is assigned either the X or the O symbol:
+* Players take turns filing the spaces in 3X3 grid.
+* The player who succeeds in placing three of their marks in the horizontal, vertical, or diagonal or wins the game.
+
+the following example game is won by the first player, X:
+![example](https://upload.wikimedia.org/wikipedia/commons/thumb/1/1b/Tic-tac-toe-game-1.svg/1280px-Tic-tac-toe-game-1.svg.png)
+
+### Project Structure
+
+```bash 
+├── README.md
+├── bin
+│   └── main.rb
+├── lib
+     ├── player.rb
+     ├── game_logic.rb
+
+```
+
+## Prerequisites
+-  Text editor
+-  Github profile
+-  Git and Ruby.
+-  Rubocop
+
+## Installations
+
+- Firstly, in order to run this on your local machine, you are going to need ruby already setup on your machine, so click this [link](https://rubyinstaller.org/) if you don't have ruby already installed and download for your specific machine.
+
+## How to Run the Game
+
+
+
+```
+$ git clone https://github.com/Yors-git/tic-tac-toe
+```
+
+- Open your git bash and clone the repo.
+
+- go to the directory of the file and run ruby "main.rb" or "main"
+
+- Good job you are now playing Tic-Tac-Toe game. 👌🙌
```

Comment:
` go to the directory of the file`: Which directory of which file.

Instead of all that directly add
```
cd tic-tac-toe
ruby bin/main.rb | ./bin/main
```